### PR TITLE
for Issue 2473 

### LINKED ISSUE #2473: Issue with running scrapy spider from script.
Hi, I'm trying to run scrapy from a script like this:

    import scrapy
    from scrapy.crawler import CrawlerProcess

    class MySpider(scrapy.Spider):
        name = "basic"
        allowed_domains = ["web"]
        start_urls = ['http://www.example.com']

        def parse(self, response):
            l = ItemLoader(item=PropertiesItem(), response = response)
            l.add_xpath('title', '//h1[1]/text()')

            return l.load_item()
    process = CrawlerProcess({
        'USER_AGENT': 'Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 5.1)'
    })

    process.crawl(MySpider)
    process.start()

However, when I run this script I get the following error:

    File "/Library/Python/2.7/site-packages/Twisted-16.7.0rc1-py2.7-macosx-10.11-
    intel.egg/twisted/internet/_sslverify.py", line 38, in <module>
    TLSVersion.TLSv1_1: SSL.OP_NO_TLSv1_1,
    AttributeError: 'module' object has no attribute 'OP_NO_TLSv1_1'

Does anyone know how to fix this? Thanks in advance.

### PR DESCRIPTION
**Pull request by Kunal614**
Fixes #2473

### CODE REVIEW

**Comment by elacuesta** [comment]
Hello @Kunal614, could you elaborate on this? I see several unrelated changes, and I'm failing to see how this patch fixes the mentioned issue.

**Comment by Kunal614** [comment]
@elacuesta nothing more was changed , only that pipeline format was changed and instead of www.example.com , i had used another web site and instead of xpath i used css selector only that much.

**Comment by Gallaecio** [comment]
@Kunal614 Please have a look at your changes in the [**Files**](https://github.com/scrapy/scrapy/pull/4391/files) tab. I think you may have included undesired files and changes.

File: News searcher from Times of india [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        
+        
+
+
+url="https://timesofindia.indiatimes.com/india/"
+ua={"UserAgent":'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:69.0) Gecko/20100101 Firefox/69.0'}
+num=input("Enter the state name or city name to get news : ")
+url+=num
+url1="https://timesofindia.indiatimes.com/india/"
+news(url,ua,url1)
```
Comment (nyov) [inline News searcher from Times of india]: This is not a BeautifulSoup project repository

File: scrapy/commands/parse.py [hunk 2]
```diff
@@ -114,24 +114,23 @@ def print_results(self, opts):
 
     def run_callback(self, response, callback, cb_kwargs=None):
         cb_kwargs = cb_kwargs or {}
-        items, requests = [], []
+        items =[]
+        request1 =[]
```
Comment (nyov) [inline scrapy/commands/parse.py]: This is a no-op change

File: scrapy/commands/parse.py [hunk 3]
```diff
@@ -114,24 +114,23 @@ def print_results(self, opts):
...
 
         for x in iterate_spider_output(callback(response, **cb_kwargs)):
             if isinstance(x, (BaseItem, dict)):
                 items.append(x)
             elif isinstance(x, Request):
-                requests.append(x)
-        return items, requests
+                request1.append(x)
+        return items, request1
```
Comment (nyov) [inline scrapy/commands/parse.py]: ... as is this.

File: scrapy/commands/parse.py [hunk 4]
```diff
@@ -114,24 +114,23 @@ def print_results(self, opts):
...
     def get_callback_from_rules(self, spider, response):
         if getattr(spider, 'rules', None):
             for rule in spider.rules:
                 if rule.link_extractor.matches(response.url):
                     return rule.callback or "parse"
         else:
-            logger.error('No CrawlSpider rules found in spider %(spider)r, '
-                         'please specify a callback to use for parsing',
-                         {'spider': spider.name})
+            logger.error('No CrawlSpider rules found in spider %(spider)r,'please specify a callback to use for parsing'{'spider': spider.name})
```
Comment (nyov) [inline scrapy/commands/parse.py]: and this.

**Comment by Gallaecio** [comment]
@Kunal614 I haven’t had a deep look, but for example, you have created what looks like a Python file in the root of the project folder, called `News searcher from Times of india` without an extension. Is that intended?

**Comment by wRAR** [comment]
Hi @Kunal614 , this PR looks like a combination of adding unrelated files and making random changes to some Scrapy files, it doesn't look related to #2473 (which is about Twisted and OpenSSL compatibility) at all, same about your comment there. So I'm closing this.

**Pull request closed without merging**